### ISSUE TITLE
Jumps to beggining of media

### ISSUE THREAD

**Issue by invinciblevenom** [TeamNewPipe/NewPipe#6472]
### Checklist

- [x] I am using the latest version - 0.21.4 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Actual behaviour

When I switch to Background Play in middle while watching a video, many times it starts playing from the very beginning. Can't take any logs for same
Well, the bug is not consistent. Sometimes, the videos play just as expected. Sometimes, from a couple of seconds ago and sometimes from the beginning. Thanks to @SameenAhnaf for this
Be it in normal, pop-up or background play, pausing a video makes it un-resumeable and changing between these methods makes videos start from the beginning again. Thanks to @ShrimpBFM for this.

### Expected behavior

The video should resume from the same duration even if we switch between different modes.

### Screenshots/Screen recordings

https://user-images.githubusercontent.com/37961534/121629843-aa3f1500-ca99-11eb-9fad-4543b63d5c8d.mp4

### Device info

 - Android version/Custom ROM version: Android 11
 - Device model: Samsung Galaxy M21

**Comment by pokegamer5547** [comment]
I noticed same issue randomly

**Comment by invinciblevenom** [comment]
The issues persists even in the latest build which is 0.21.8

**Comment by infinitewaveparticle** [comment]
Yes this issue is quite annoying and persistent. Please fix it for the next release!

**Comment by IJohnDoeI** [comment]
This happens with me too.

**Comment by Flam3z** [comment]
Issue also happens to me.

0.21.10 | Android 11 | Galaxy S10
